webpacker_guide.md

```diff
@@ -14,16 +14,75 @@ After reading this guide, you will know:
 
 ## What Is Webpacker?
 
+Webpacker is a Rails wrapper around the [webpack](https://webpack.js.org) build system that provides a standard webpack configuration and reasonable defaults. 
 
+### What is webpack?
+
+The goal of webpack, or any front-end build system, is to allow you to write your front-end code in a way that is convenient for developers and then package that code in a way that is convenient for browsers. With webpack, you can manage JavaScript, CSS, and static assets like files or fonts. Webpack will allow you to write your code, reference other code in your application, transform you code, and combine your code into easily downloadable packs.
 
 ## How is Webpacker Different from Sprockets?
 
+Rails also ships with Sprockets, an asset-packaging tool whose features overlap with Webpacker. Both tools will compile your JavaScript into into browser-friendly files, and minify and fingerprint them in production. Both tools allow you to incrementally change files in development.
+
+Sprockets, which was designed to be used with Rails, is somewhat simpler to integrate. In particular, code can be added to Sprockets via a Ruby gem. However, webpack is better at integrating with more current JavaScript tools and NPM packages, and allows for a wider range of integration.
+
+You should choose webpacker over Sprockets on a new project, if you want to use NPM packages, and if you want access to the most current JavaScript features and tools. You should choose Sprockets over Webpacker for legacy applications where migration might be costly, if you want to integrate using Gems, or if you have a very small amount of code to package. 
+
```

Comment:
Is this a Webpacker specific guide, or an overall Rails JS guide? 

If the latter, would suggest a note that Webpack is the recommended JS option for Rails 6+, but also provide guidance on how to revert to Sprockets for JS, eg. 
https://stackoverflow.com/questions/55213868/rails-6-how-to-disable-webpack-and-use-sprockets-instead

Reply:
Webpacker for now, trying to keep this focused in the hope of getting it done